rust/actix-web/src/main.rs

```diff
@@ -0,0 +1,29 @@
+/// An actix-web 3 application that raises an error when visiting http://127.0.0.1:3001/.
+use std::io;
+
+use actix_web::{get, App, Error, HttpRequest, HttpServer};
+
+#[get("/")]
+async fn failing(_req: HttpRequest) -> Result<String, Error> {
+    Err(io::Error::new(io::ErrorKind::Other, "An error happens here").into())
+}
+
+#[actix_web::main]
+async fn main() -> io::Result<()> {
+    let _guard =
+        sentry::init("https://<PUBLIC_DSN_KEY>:<PRIVATE_DSN_KEY>@sentry.io/<PROJECT_ID>");
```

Comment:
private key part is deprecated:

```suggestion
        sentry::init("https://<KEY>@sentry.io/<PROJECT_ID>");
```